NwPluginAPI/Loader/AssemblyLoader.cs

```diff
@@ -226,5 +239,56 @@ private static bool TryGetAssembly(string path, out Assembly assembly)
 			assembly = null;
 			return false;
 		}
+		
+		/// <summary>
+		/// Attempts to load Embedded assemblies (compressed) from the target
+		/// </summary>
+		/// <param name="target">Assembly to check for embedded assemblies</param>
+		private static void ResolveAssemblyEmbeddedResources(Assembly target)
+		{
+			Log.Debug($"Attempting to load embedded resources for {target.FullName}", Log.DebugMode);
+
+			
+			var resourceNames = target.GetManifestResourceNames();
+			foreach (var name in resourceNames)
+			{
+				Log.Debug($"Found {name}", Log.DebugMode);
+				if (name.EndsWith(".dll"))
```

Comment:
```suggestion
				if (name.EndsWith(".dll", StringComparison.OrdinalIgnoreCase))
```

Maybe let's use `StringComparison.OrdinalIgnoreCase` to avoid potential issues?